### ISSUE TITLE
Are YOLOv4-P5 (896x896) and YOLOv4-P6 (1280x1280) described in the Scaled-YOLOv4 paper supported? If not, is it possible to add those?

### ISSUE THREAD

**Issue by xjinai** [AlexeyAB/darknet#7414]
Are YOLOv4-P5 (896x896) and YOLOv4-P6 (1280x1280) described in the Scaled-YOLOv4 paper supported? If so, could you point me to the cfg and pre-trained weights files for YOLOv4-P5 and YOLOv4-P6? If not, is it possible to add the support of those?

**Comment by stephanecharette** [comment]
@AlexeyAB For those of us like myself only familiar with YOLOv4.cfg and YOLOv4-tiny.cfg, what is YOLOv4-p5.cfg and YOLOv4-p6.cfg you've recently added?

**Comment by AlexeyAB** [comment]
@xjinai @AlexeyAB 

**For Detection** - use the same as usual:
```
./darknet detector test cfg/coco.data cfg/yolov4-p5.cfg yolov4-p5.weights -ext_output dog.jpg
./darknet detector test cfg/coco.data cfg/yolov4-p6.cfg yolov4-p6.weights -ext_output dog.jpg
```

You can download pre-trained weights on COCO:
* for `yolov4-p5.cfg` https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-p5.weights
* for `yolov4-p6.cfg` https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-p6.weights


----

**For Training** - change these lines before each of 3 for p5 (of 4 for p6) `[yolo]`-layers:
https://github.com/AlexeyAB/darknet/blob/9a86fce494b1d82b774d36be76747fcb58f81aa4/cfg/yolov4-p5.cfg#L1810-L1811
`filters=<(5 + num_classes) x 4>`
`activation=logistic` - for **training and detection by using Darknet**: https://github.com/AlexeyAB/darknet
`activation=linear` - for **training and detection by using Pytorch** Scaled-YOLOv4 (CSP-branch): https://github.com/WongKinYiu/ScaledYOLOv4/tree/yolov4-csp

For training use pre-trained weights:
* for `yolov4-p5.cfg` https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-p5.conv.232
* for `yolov4-p6.cfg` https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-p6.conv.289

Currently Pytorch is more suitable for training on multiple-GPUs.

----

**Accuracy - Speed:**

* Scaled-YOLOv4-P6 - 54.5% AP (COCO) - 30 FPS
* YOLOR-P6 - 52.6% AP (COCO) - 49 FPS

Scaled-YOLOv4-P6 is slower, but +2% more accurate than YOLOR-P6 that is the best in terms of speed/accuracy for Waymo autonomous driving challenge: https://github.com/AlexeyAB/darknet/issues/7828

<img width="687" alt="123036148-3e43a180-d3f5-11eb-926d-bbc810f0ea6a" src="https://user-images.githubusercontent.com/4096485/123530901-460e8900-d708-11eb-8f3a-7a6a0e44acec.png">



----

Speed and accuracy on COCO is validated by using Pytorch: https://github.com/WongKinYiu/ScaledYOLOv4/tree/yolov4-csp
![112776361-281d8380-9048-11eb-8083-8728b12dcd55 (1)](https://user-images.githubusercontent.com/4096485/123530761-ba482d00-d706-11eb-858b-2c64e04df78b.png)
